BUILDING.md

```diff
@@ -44,21 +44,21 @@ Start a console, with administrator privilege, and type:
 
 `choco install bazel --version 2.0.0`
 
-In the same console, install Python 2.7 and MSYS2 as well:
+In the same console, install Python and MSYS2 as well:
 
-`choco install python2`
+`choco install python`  
```

Comment:
Would it be better to be specific and require python3 here?

Reply:
`python` is a specific [chocolatey package](https://community.chocolatey.org/packages/python/3.9.4) that I have successfully tested. `python3` is a different [package](https://community.chocolatey.org/packages/python3/3.9.4) that should be tested with building AGI before we reference it, so for now I would prefer to keep `python`.